ui/site/src/component/socket.ts

```diff
@@ -240,7 +240,7 @@ export default class StrongSocket {
         return;
       }
       // it's impossible but according to previous logging, it happens nonetheless
-      if (m.v > this.version + 1) return reload();
+      if (m.v > this.version + 1 && !document.body.getAttribute('data-kid')) return reload();
```

Comment:
if in 'kid' mode there is already a class 'kid' on the body so we could just check that, instead of adding a new datakid attribute?

Reply:
ah yes just using document.body.getClass ?

Reply:
or hasClass("kid") maybe, not sure to remember correctly the way to get it

Reply:
you can use document.body.classList.contains('kid')